src/main/java/com/hierynomus/msfscc/fileinformation/FileInformationFactory.java

```diff
@@ -107,6 +107,25 @@ public void write(FileBasicInformation info, Buffer outputBuffer) {
         decoders.put(FileBasicInformation.class, basicCodec);
         encoders.put(FileBasicInformation.class, basicCodec);
 
+        FileInformation.Codec<FileDispositionInformation> dispositionCodec = new FileInformation.Codec<FileDispositionInformation>() {
+            @Override
+            public FileInformationClass getInformationClass() {
+                return FileInformationClass.FileDispositionInformation;
+            }
+
+            @Override
+            public FileDispositionInformation read(Buffer inputBuffer) throws Buffer.BufferException {
+                throw new UnsupportedOperationException();
+            }
+
+            @Override
+            public void write(FileDispositionInformation info, Buffer outputBuffer) {
+                outputBuffer.putRawBytes(FileInformationFactory.getFileDispositionInfo(info.isDeleteOnClose()));
```

Comment:
Refactor the `getDispositionInfo` method to use the same naming and parameter convention as the other methods.